packages/react/src/components/F0Card/CardInternal.tsx

```diff
@@ -103,149 +111,207 @@ export interface CardInternalProps {
    * Private prop
    */
   forceVerticalMetadata?: boolean
+
+  /**
+   * Optional drag configuration to make the card draggable
+   */
+  drag?: {
```

Comment:
why not use an external wrapper to decouple the card from drag as the dragging logic is not related to the card